test/fc_mpi/src/CMakeLists.txt

```diff
@@ -1,5 +1,10 @@
 if (MPI_FOUND)
     add_executable(example example.f90)
+    if (CMAKE_SYSTEM_NAME MATCHES "Windows")
+        # path in appveyor.yml
+        #target_link_libraries(example "C:\software\msmpi\libmsmpi64.a")
+        target_link_libraries(example "C:/software/msmpi/libmsmpi64.a")
```

Comment:
Ehm, this is not smart solution  - the hardcoded path is for the *appveyor.yml* build.